### PR TITLE
Fallback to placeholder logo in search page.

### LINKED ISSUE #700: Missing images are not correctly handled on search results
Search results are sometimes broken, see image.
We should handle them in the same way we do in the other pages.

Schermata da 2020-09-18 17-31-45

### PR DESCRIPTION
**Pull request by bfabio**
When the software logo is not available in the search page,
fallback to a default image.

Fix #700.

### CODE REVIEW

**Comment by bfabio** [comment]
Disclaimer: this sucks but I don't think we can do much better without fixing #695 first.

**Comment by libremente** [comment]
> Disclaimer: this sucks but I don't think we can do much better without fixing #695 first.

I know and we discussed this many times with @sebbalex. Let's merge this now so we fix the issue and then in the refactoring we will see how to properly handle every case.

File: assets/js/esDevelopersItaliaQuery.js [hunk 1]
```diff
@@ -964,30 +966,31 @@ esDevelopersItaliaQuery.prototype.renderSoftware = function (software) {
   var category_id = this.getSoftwareType(software);  // generic-sw vs public-sw
   var description = this.languageFallback(software.publiccode.description);
 
+  const placeholder = (category_id == 'public-sw')
+    ? '/assets/images/cover_softwareriuso.png'
```
Comment (libremente) [inline assets/js/esDevelopersItaliaQuery.js:970]: In another PR we should really change this filename to `software_riuso` and fix all occurrences, perfect for hacktoberfest :smile:

Comment (bfabio) [reply assets/js/esDevelopersItaliaQuery.js:970]: Agreed :wink:

**Review by sebbalex — APPROVED** [review]
I think this could be ok for ther moment, waiting for #695

**Pull request merged**